core/src/calleeFunctions/helpers/uniswapV3.ts

```diff
@@ -66,3 +66,17 @@ export const convertCollateralToDai = async function (
     const daiAmount = new BigNumber(daiIntegerAmount._hex).shiftedBy(-DAI_NUMBER_OF_DIGITS);
     return daiAmount;
 };
+
+export const convertDaiToMkr = async function (network: string, amountDai: BigNumber): Promise<BigNumber> {
+    const auctionIntegerAmount = amountDai.shiftedBy(DAI_NUMBER_OF_DIGITS).toFixed(0);
+    const uniswapV3quoterContract = await getUniswapV3quoterContract(network);
+    const daiIntegerAmount = await uniswapV3quoterContract.callStatic.quoteExactInputSingle(
+        await getContractAddressByName(network, 'MCD_DAI'),
```

Comment:
I think `getTokenAddressByNetworkAndSymbol` is the better function for that purpose